server/src/main/java/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -827,6 +829,10 @@ private Pair<List<UserAccountJoinVO>, Integer> getUserListInternal(Account calle
             }
         }
 
+        if (userSource != null) {
+            sc.setParameters("userSource", userSource.toString());
+        }
```

Comment:
We could use setParametersIfNotNull() here

Reply:
If the `setParametersIfNotNull` method were used, scenarios where `userSource` is `null` would result in an NPE because the `toString` method of the enum is invoked.